README.md

```diff
@@ -114,10 +114,6 @@ See [SECURITY.md](SECURITY.md) for more details.
 See
 [all contributors](https://github.com/strongloop/loopback-next/graphs/contributors).
```

Comment:
I think this line is no longer necessary since "all contributors" section is following immediately.